AdminWebsite/AdminWebsite.UnitTests/Helper/AdministratorRoleClaimsHelperTest.cs

```diff
@@ -0,0 +1,91 @@
+﻿using AdminWebsite.Helper;
+using AdminWebsite.Services.Models;
+using FluentAssertions;
+using NUnit.Framework;
+using System.Collections.Generic;
+using System.Linq;
+using System.Security.Claims;
+
+namespace AdminWebsite.UnitTests.Helper
+{
+    public class AdministratorRoleClaimsHelperTest
+    {
+        [Test]
+        public void GetAdministratorClaims_Returns_Claims()
+        {
+            var userGroupData = new UserGroupData
+            {
+                UserRole = "Role1",
+                CaseTypes = new List<string> { "Case1", "Case2" }
+            };
+
+            var claims = new AdministratorRoleClaimsHelper(userGroupData).GetAdministratorClaims().ToList();
+
+            claims.Should().NotBeNull();
+            claims.Should().NotContainNulls();
+            claims.Should().HaveCountGreaterThan(0);
+            var userCaseTypesClaim = claims.FirstOrDefault(x => x.Type == "UserCaseTypes");
+            userCaseTypesClaim.Should().NotBeNull();
+            userCaseTypesClaim?.Value.Should().Be("Case1,Case2");
+        }
+
+        [TestCase("VhOfficer", true)]
+        [TestCase("SomeRole", false)]
+        public void IsVhOfficerAdministratorRole_Properties_Are_Correctly_Set(string role, bool expectedClaimValue)
```

Comment:
a bit hard to read this, both in terms of what the test is supposed to test and what it actually does. it looks a bit like you're testing two things

Reply:
Maybe, but you have to read it as its running 2 unit tests, 1. role = VhOfficer, 2 = SomeRole.
Reduces duplication. its like specflow examples, running the test multiple time for different criteria.